### ISSUE TITLE
Floating menu mobile chrome

### ISSUE THREAD

**Issue by thousandsofraccoons** [ueberdosis/tiptap#1994]
### What’s the bug you are facing?

Floating menu hides only after entering space character.
![IMG_20211008_114547](https://user-images.githubusercontent.com/33694447/136500217-f326a022-37c6-4dbc-a429-f360e099b384.jpg)

### How can we reproduce the bug on our side?

Open example with floating menu on mobile, enter anything

### What did you expect to happen?

Floating menu gotta close

### Did you update your dependencies?

- [X] Yes, I’ve updated my dependencies to use the latest version of all packages.

### Are you sponsoring us?

- [X] Yes, I’m a sponsor. 💖

**Comment by dkrym** [comment]
I experienced this issues and would like to provide more info.

- it seems this is problem only on Android, IOS works ok for me
- after some investigation I found out the problem is that by default Android use Spell Check (signaled by underline when you type the word
  - when you are in the spell check mode (underline), floating menu does not register typing (or updates at all?)
  - I thought the reason is the word is not entered until space and therefor typing not registered but no. Using collab function I can see every letter is actually sent, just floating menu does not register updates until space

I have no idea how to fix as I actually don't understand why is this exactly a problem.

Not a fix, but workaround which works for now:
- Disable Spell Check on your phone (not really practical)
- add `autocorrect="off"` to disable autocorrect on your editor which disables the android spell check so it works (but you lose spell check obviously)
  - I tested it by adding it to editor but it could be added only to empty paragraphs etc

![btn](https://user-images.githubusercontent.com/9148471/139771923-efcaaee9-fada-470d-90be-5ae0495f18b0.gif)
